.github/scripts/changelog_manager.py

```
@@ -0,0 +1,140 @@
+
+    def generate_md(self):
+        print("CHANGELOG.md 생성 시작...")
+        data = self._load_json()
+        
+        md_content = ["# CHANGELOG\n\n이 파일은 워크플로우에 의해 자동으로 생성됩니다.\n"]
+
+        for release in data.get("releases", []):
+            version = release.get("version", "N/A")
+            # 타임스탬프에서 날짜만 추출
+            date = datetime.fromisoformat(release.get("timestamp").replace("Z", "+00:00")).strftime("%Y-%m-%d")
+            pr_number = release.get("pr_number", "N/A")
```

Ruff (0.12.2)

Local variable `pr_number` is assigned to but never used

Remove assignment to unused variable `pr_number`

(F841)